.travis/script.sh

```diff
@@ -145,14 +145,24 @@ build_cc2650() {
 }
 
 build_cc2652() {
+    CMAKE_FLAGS="                        \
+        -DOT_COMPILE_WARNING_AS_ERROR=on \
```

Comment:
These `on`s get turned into `1`s for the preprocessor correct?

Reply:
Yes, see [etc/cmake/options.cmake](https://github.com/openthread/openthread/blob/master/etc/cmake/options.cmake).